src/service-override/search.ts

```diff
@@ -14,6 +16,7 @@ export default function getServiceOverride (): IEditorOverrideServices {
     [ISearchService.toString()]: new SyncDescriptor(SearchService, [], true),
     [ISearchViewModelWorkbenchService.toString()]: new SyncDescriptor(SearchViewModelWorkbenchService, [], true),
     [ISearchHistoryService.toString()]: new SyncDescriptor(SearchHistoryService, [], true),
-    [IReplaceService.toString()]: new SyncDescriptor(ReplaceService, [], true)
+    [IReplaceService.toString()]: new SyncDescriptor(ReplaceService, [], true),
+    [IProgressService.toString()]: new SyncDescriptor(ProgressService, [], true)
```

Comment:
I'm not sure it makes sense to have it here since this service-override already required the view service override which includes it (And referencing it from both service override forces the code to be in monaco-vscode-api instead of the service override lib)

Reply:
Ah, 🤦 . Completely true. Updating now...